.lando.yml

```diff
@@ -0,0 +1,27 @@
+name: amphtml
```

Comment:
Assuming this file allows comments, maybe post a comment at the top that states the purpose of this file with a link to https://docs.lando.dev/config/lando.html?

Reply:
Added in 77f9c18